wp-content/themes/engage/assets/scss/templates/_homepage.scss

```diff
@@ -19,6 +19,19 @@ $lightGrey: #fbfbfb;
 			grid-column-end: span 3;
 		}
 
+		@supports not (display: grid) {
+			display: flex;
+			flex: 0 1 23.6%;
+	
+			@media screen and (max-width: 1200px) {
```

Comment:
@Lay-ton @NWebb5 One thing to notice is we should use the media query mixins (`@include media($mobile) {}` ext.) for media queries. I made a big comment in the `_mixins.scss` file of which I will be committing Monday :)

Reply:
Got it! I look forward to seeing that commit!